docs/pages/customers/oregon.md

```diff
@@ -0,0 +1,86 @@
+  />
+
+**Partner Since: 2021**
+
+## Overview
+
+[Oregon State University](https://oregonstate.edu/) is a premier public land-grant research university located in Corvallis, Oregon, recognized for innovation, world-class research, and a diverse academic environment with strong community engagement and global impact.
+To improve their course planning and faculty assignment, they developed Resource Insights-an application, which balances course sections, faculty availability, and student demand.
+It replaced outdated Excel-based methods with a modern online system.
+The tool integrates with existing university systems to provide real-time enrollment data, eliminating manual data entry.
+
+<span class="only-light-mode">
+<blockquote style="margin: 32px 0; padding: 24px 32px; background: #f5f5f7; border-left: 6px solid var(--muidocs-palette-primary-main); border-radius: 8px; font-size: 1.15rem; font-style: italic;">
+  MUI X provides the right balance of performance, customization, and React integration. If you need a scalable, spreadsheet-like experience in your web app, it's definitely worth considering.
+  <br>
+  <span style="display: block; margin-top: 12px; font-size: 1rem; font-style: normal; color: #555; font-weight: 500;">
+    – Mark Clements, Manager, Software Development & Quality
+  </span>
+</blockquote>
+</span>
+<span class="only-dark-mode">
+<blockquote style="margin: 32px 0; padding: 24px 32px; background: var(--muidocs-palette-background-default); border-left: 6px solid var(--muidocs-palette-primary-main); color: #f5f5f7; border-radius: 8px; font-size: 1.15rem; font-style: italic;">
+  MUI X provides the right balance of performance, customization, and React integration. If you need a scalable, spreadsheet-like experience in your web app, it's definitely worth considering.
+  <br>
+  <span style="display: block; margin-top: 12px; font-size: 1rem; font-style: normal; color: #bbb; font-weight: 500;">
+    – Mark Clements, Manager, Software Development & Quality
+  </span>
+</blockquote>
+</span>
+
+## Challenge
+
+The university needed a robust solution to handle tabular data efficiently.
+Planners and administrators required a system that could provide spreadsheet-like interactions, allowing them to filter, sort, and manipulate data dynamically.
+Key challenges included:
+
+- Lack of real-time data integration in current processes.
+- The need for an Excel-like experience in a web-based tool.
+- Finding a framework that was well-supported, actively maintained, and built specifically for React.
+- Ensuring high performance and customization while keeping the UI intuitive.
+  While alternatives like AG Grid were considered, MUI X was preferred because it's deeply integrated with React, whereas AG Grid's approach spans multiple frameworks, leading to inconsistencies in React-specific implementations.
```

Comment:
```suggestion
  While alternatives like AG Grid were considered, MUI X was preferred because it's deeply integrated with React, whereas AG Grid's approach spans multiple frontend frameworks, leading to inconsistencies in React implementations.
```

Removed "specific", as the point is that AG Grid isn't React-specific.